modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -84,7 +84,10 @@ where:
 . Optional: You can customize the following settings for OVN-Kubernetes to meet your network infrastructure requirements:
 +
 --
-* Maximum transmission unit (MTU)
+* Maximum transmission unit (MTU). Consider the following prior to customizing the MTU for this optional step:
+** If you have the default MTU applied, and you want to keep the default MTU during migration, this step can be ignored. 
+** If you have a custom MTU applied, and you want to keep the custom MTU during migration, you must declare the value in this step. 
```

Comment:
@stevsmit How about updating it to say, 'You must declare the custom MTU value in this step'?